lambda_functions/v2/functions/csv-forwarder/main.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+    "github.com/aws/aws-lambda-go/events"
```

Comment:
These indents seem to be using spaces rather than tabs should be updated to tabs as that's Go standard.

Are you using VSCode for this file? I'd of thought the IDE would fix a few of these issues for you if you were using GoLand

Reply:
Was using GoLand until my trial expired, and had to switch out to something else.
Will get VSCode and try sort out the spacing.